.github/workflows/playwright.yml

```diff
@@ -150,22 +156,16 @@ jobs:
         if: ${{matrix.testSuite !='elements-regression'}}
         env:
           TEST_SUITE: "@${{matrix.testSuite}}"
-        run: npm run test:playwright -- --reporter=${{inputs.reporter || 'list'}}
+        run: npm run test:playwright -- --reporter=${{inputs.reporter || 'list,github'}}
       - name: Run element regression tests
         if: ${{matrix.testSuite =='elements-regression'}}
-        run: npm run test:playwright:elements-regression -- --reporter=${{inputs.reporter || 'list'}}
+        run: npm run test:playwright:elements-regression -- --reporter=${{inputs.reporter || 'list,github'}}
       - uses: actions/upload-artifact@v3
-        if: always()
+        if: ${{ inputs.fail_fast == false || failure() }}
```

Comment:
Why not upload the results anyhow? If we don't have these its hard to debug

Reply:
```
(https://github.com/elementor/elementor/actions/runs/7685951345/job/20944299627#step:14:8)
No files were found with the provided path: test-results/. No artifacts will be uploaded.
```
Fixed. But we will have this warnings